pkg/apis/componentconfig/types.go

```diff
@@ -356,7 +354,7 @@ type KubeControllerManagerConfiguration struct {
 	// DEPRECATED: deletingPodsBurst is the number of nodes on which pods are bursty deleted in
 	// case of node failure. For more details look into RateLimiter.
 	DeletingPodsBurst int32
-	// nodeMontiorGracePeriod is the amount of time which we allow a running node to be
+	// nodeMonitorGracePeriod is the amount of time which we allow a running node to be
```

Comment:
s/nodeMonitorGracePeriod/NodeMonitorGracePeriod 

Most of the field comments in this struct aren't consistent with the field, e.g. incorrect case sensitive of initial of the field. Since you're here, please check and fix all of them. Thanks!

Reply:
@xiangpengzhao 👌